src/features/auth/ui/SignInSocialLogin.tsx

```
@@ -0,0 +1,45 @@
+'use client';
+import useSocialLogin from '../hooks/useSocialLogin';
+
+const SignInSocialLogin = () => {
+  const { handleSocialLogin } = useSocialLogin();
+  return (
+    <div className="space-y-3">
+      <button
+        className="w-full flex items-center justify-center py-3 px-4 bg-gray-800/50 hover:bg-white/5 border border-gray-700 rounded-[10px] text-white transition-all duration-200"
+        onClick={() => handleSocialLogin('google')}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)